src/lights/SphericalHarmonicsLight.js

```diff
@@ -0,0 +1,37 @@
+import { Light } from './Light.js';
+
+/**
+ * @author Andreas Atteneder
+ */
+
+function SphericalHarmonicsLight( sphericalHarmonics3, intensity = 1.0 ) {
+
+	Light.call( this );
+
+	this.sphericalHarmonics3 = sphericalHarmonics3;
+	this.intensity = intensity;
+
+	this.type = 'SphericalHarmonicsLight';
+
+	this.castShadow = undefined;
+
+}
+
+SphericalHarmonicsLight.prototype = Object.assign( Object.create( Light.prototype ), {
+
+	constructor: SphericalHarmonicsLight,
+
+	isSphericalHarmonicsLight: true,
+
+	getCoefficients: function() {
```

Comment:
I'd promote this to SphericalHarmonics3 class as the function "toArray" or similar.